README.md

```diff
@@ -1,2 +1,80 @@
-# talent.learnersguild.org
+# Talent
 
+Talent is an app used for employers, future, past, and present learners, and anyone interested in Learner's Guild to see what our learners have accomplished!
+
+## Getting Started
+
+Fork the Talent repo.
+
+Clone your forked repo.
+
+Set your remote upstream to the original repo.
+
+### Prerequisites
+
+You will need, and need to know, React.
+
+- [React](https://reactjs.org/tutorial/tutorial.html)
+- [React Router](https://reacttraining.com/react-router/web/api/Redirect)
+
+Knowing about Webpack and Babel would help as well.
+- [Webpack](https://www.youtube.com/watch?v=JdGnYNtuEtE&index=1&list=PLkEZWD8wbltnRp6nRR8kv97RbpcUdNawY)
+- [Babel](http://babeljs.io/docs/setup/)
+
+### Installing
+
+Install the modules.
+
+```
+npm install
+```
+
+Then run the server.
+
+```
+npm start
+```
+Globally install nvm, avn, and avn-nvm.
+
+```
+curl -o- https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+npm install -g avn avn-nvm
+avn setup
+```
+
+## FUTURE TESTING SUBSECTION
+
+## Built With
+
+* [React](https://reactjs.org/) - Library used to build the app.
+* [React Router](https://reacttraining.com/react-router/) - Library used to give React routing capabilities.
+* [Webpack](https://webpack.js.org/) - Library used to bundle js, css, and html.
+* [Babel](https://babeljs.io/) - Library used to compile js.
+* [Sass](http://sass-lang.com/) - Library used to improve css.
+
+## Versioning
+
+express - 4.16.2
+node - 8.1.2
+react - 16.0.0
+
+## Authors
+
+* **Jared Grippe** - *Initial work* - [Jared Grippe](https://github.com/deadlyicon)
```

Comment:
Can we just keep the Names and links of the contributors here.

Reply:
Okay.